regen/op_private

```diff
@@ -606,6 +606,12 @@ addbits('entersub',
     # 7    => OPpLVAL_INTRO, already defined above
 );
 
+# these builtin:: implementing OPs also use dXSTARG and expect HASTARG to be
+# set if a targ is available
+addbits($_,
+        2      => qw(OPpENTERSUB_HASTARG  TARG  ),
+) for qw(refaddr reftype ceil floor goto);
```

Comment:
Did you mean to include `goto` in here as well? I don't see any other changes elsewhere that would include it.

Reply:
We check for the flag when the XSUB is called from goto, so while we never set the flag it is reserved.

Reply:
Ahyes, makes sense. Might be worth adding a comment to that effect then.

Reply:
I thought of renaming `OPpENTERSUB_HASTARG` to `OPpXSUB_HASTARG` to make things a bit clearer but it is used in code generated by EU::PXS and by code in PathTools, so I suspect the rename would be more confusing than helpful.

Reply:
added a more detailed comment